src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,15 +4,18 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
+
     @Override
     public Order completeOrder(Long userId) {
-        // TODO: add log message about method completeOrder was called
+        logger.debug("completeOrder method was called.");
```

Comment:
The log message in the 'completeOrder' method is not informative enough. It would be more helpful to include the 'userId' to track which user's order is being completed. For example, logger.debug("completeOrder method was called with userId={}", userId);